spec/models/taxonomy/expanded_taxonomy_spec.rb

```diff
@@ -40,8 +46,44 @@ def home_page
   let(:bramley) { fake_taxon("Bramley") }
   let(:cox) { fake_taxon("Cox") }
 
+  describe 'A child and parent taxon not attached to the root taxon' do
+    before :each do
+      @rootless_parent = fake_taxon("Rootless Parent")
+      @rootless_child = fake_taxon("Rootless Child")
+
+      publishing_api_has_item(@rootless_parent)
+      publishing_api_has_item(@rootless_child)
+
+      publishing_api_has_expanded_links(
+        content_id: @rootless_parent["content_id"],
+        expanded_links: {
+          child_taxons: [@rootless_child],
+        },
+      )
+
+      publishing_api_has_expanded_links(
+        content_id: @rootless_child["content_id"],
+        expanded_links: {
+          parent_taxons: [@rootless_parent],
+        },
+      )
+    end
+
+    describe 'Rootless taxonomies' do
+      it 'has one parent, which has no root' do
+        taxonomy = Taxonomy::ExpandedTaxonomy.new(@rootless_child['content_id']).build
+        expect(taxonomy.parent_expansion.children.map(&:internal_name)).to match_array(['i-Rootless Parent'])
```

Comment:
It took me a while to figure out what `'i-Rootless Parent'` is, it might be clearer to say `@rootless_parent.dig('details', 'internal_name')` (or whatever the correct code is), and perhaps easier to compare the `content_id`?

Reply:
Changed to use @rootless_child['content_id'] to compare

Reply:
👍